sdk/keyvault/keyvault-common/src/challengeBasedAuthenticationPolicy.ts

```diff
@@ -97,7 +97,7 @@ export function parseWWWAuthenticate(wwwAuthenticate: string): ParsedWWWAuthenti
   // Then we only keep the strings with an equal sign after a word and before a quote.
   // also splitting these sections by their equal sign
   const keyValues = parts.reduce<string[][]>(
-    (parts, str) => (str.match(/\w="/) ? [...parts, str.split("=")] : parts),
+    (acc, str) => (str.match(/\w="/) ? [...acc, str.split("=")] : acc),
```

Comment:
This is very arbitrary 😄

Reply:
`parts` in the closure here shadows the variable of the same name in the enclosed scope, so this seems reasonable to me.